es6-classes/circle.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super(radius);
+    this.radius = radius;
+    this.area = 3.1415926535 * (this.radius * this.radius);
+    this.perimeter = 2 * 3.1415926535 * this.radius;
+  }
+
+  describe() {
+    return `The circle's radius is ${this.radius}, the area is ${this.area}, and the perimeter is ${this.perimeter}.`;
```

Comment:
This should be calling the `describe` method of `super` to access `perimeter` & `area`